cmake/evt-core_compiler.cmake

```diff
@@ -32,6 +33,10 @@ else()
 endif()
 message(STATUS "${TARGET_DEV} targeted")
 
+option (FORCE_COLORED_OUTPUT "Always produce ANSI-colored output (GNU/Clang only)." OFF)
```

Comment:
I would argue that we would want this on by default, but I don't really care

Reply:
If gcc has access to the console then it does this by default, however a build tool like ninja pipes the output to the program so gcc disables it. This just adds the flag to force colorizing. Not everyone (for some reason) like it and something like github actions shouldn't have to deal with that.